client/src/components/Order/order.rate/RateOrder.tsx

```diff
@@ -35,27 +35,32 @@ const RateOrder: FC<RateOrderProps> = () => {
 
                 setOrder(data)
 
-            } 
-            
-            if(!data.length) {
+            } else {
 
                 alert('Current order has been already rated')
                 history.push('/')
+
             }
-            
         }
-
         readOrderForRate()
     },[])
 
 
     const onSubmit = (event: React.FormEvent<HTMLFormElement>) => {
         event.preventDefault()
 
+        const { ratedSum, ratedQuantity } = order[0].master
+        const newRatedSum = ratedSum + rating
+        const newRatedQuantity = ratedQuantity + 1
+        const newRating = Number((newRatedSum / newRatedQuantity).toFixed(2))
+
         axios.put(`/${URL.RATED_ORDER}`, {
             id: order[0].id,
-            order_rated: rating,
-            master_id: order[0].masterId
+            orderRated: rating,
+            masterId: order[0].master.id,
```

Comment:
As I know you do not need the three params of rating - duplicate data. Can ordrer[0] be undefined?